src/state/lists/utils.test.ts

```diff
@@ -0,0 +1,64 @@
+import { TokenList, VersionUpgrade } from '@uniswap/token-lists'
+
+import { shouldAcceptMinorVersionUpdate } from './utils'
+
+function buildTokenList(count: number): TokenList {
+  const tokens = []
+  for (let i = 0; i < count; i++) {
+    tokens.push({
+      name: `Token ${i}`,
+      address: `0x${i.toString().padStart(40, '0')}`,
+      symbol: `T${i}`,
+      decimals: 18,
+      chainId: 1,
+      logoURI: `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x${i
+        .toString()
+        .padStart(40, '0')}/logo.png`,
+    })
+  }
+  return {
+    name: 'Defi',
+    logoURI:
+      'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x514910771AF9Ca656af840dff83E8264EcF986CA/logo.png',
+    keywords: ['defi', 'uniswap'],
+    timestamp: '2021-03-12T00:00:00.000Z',
+    version: {
+      major: 1,
+      minor: 0,
+      patch: 0,
+    },
+    tokens,
+  }
+}
+
+describe('shouldAcceptMinorVersionUpdate', () => {
+  it('returns false for patch when tokens have changed', () => {
+    expect(
+      shouldAcceptMinorVersionUpdate('test_list', buildTokenList(1), buildTokenList(2), VersionUpgrade.PATCH)
+    ).toEqual(false)
+  })
+
+  it('returns true for patch when tokens are the same', () => {
+    expect(
```

Comment:
nit: did you run prettier? why did this break into new line?

Reply:
very weird,  the lint wasn't fixed on save when i first wrote this